DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: sparta
 Type: Package
 Title: Trend Analysis for Unstructured Data
-Version: 0.2.00
+Version: 0.2.01
 Date: 2019-04-12
```

Comment:
sorry to be a pain but can you update the date too?!
```suggestion
Date: 2019-04-25
```

Reply:
@JackHHatfield91 - I tried a new feature, looks like you can just click a button to make the suggested change, give it a go

Reply:
Nice, seems to work. Sorry, I'll try and remember to change the version and date in future